src/js/index.js

```diff
@@ -513,16 +527,31 @@ export const hpe = deepFreeze({
     },
   },
   tab: {
-    border: {
-      color: 'text-xweak',
+    color: 'text',
+    active: {
+      background: 'background-contrast',
     },
-    color: 'text-xweak',
-    margin: {
-      horizontal: 'none',
+    hover: {
+      background: 'background-contrast',
+      color: 'text',
     },
-    pad: {
-      horizontal: 'small',
+    border: {
+      side: 'bottom',
+      color: 'background-back',
+      active: {
+        color: 'border',
+      },
+      hover: {
+        color: 'background-back',
+      },
```

Comment:
Seems like these may have changed. In https://www.figma.com/file/Kp4dWyhUTnKIJ1Cg5CoL9o/HPE-Tabs-Component?node-id=208%3A0 I am seeing the following:

```suggestion
      color: 'border',
      active: {
        color: 'border-strong',
      },
      hover: {
        color: 'border',
      },
```